pkg/controller/restore_controller.go

```diff
@@ -185,17 +197,19 @@ func (controller *restoreController) processNextWorkItem() bool {
 }
 
 func (controller *restoreController) processRestore(key string) error {
-	glog.V(4).Infof("processRestore for key %q", key)
+	logContext := controller.logger.WithField("key", key)
+
+	logContext.Debug("Running processRestore")
 	ns, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		glog.V(4).Infof("error splitting key %q: %v", key, err)
+		logContext.WithField("error", err).Info("Error splitting queue key")
 		return err
 	}
 
-	glog.V(4).Infof("Getting restore %s", key)
+	logContext.Debug("Getting Restore")
```

Comment:
this changed from info to debug, was it intentional?

Reply:
Same comment as above - yes - we basically want to lump Infof/V1/V2 as Info and V3+ as Debug